woudc_data_registry/product/ozonesonde/ozonesonde_generator.py

```diff
@@ -75,9 +76,13 @@ def execute(path, bypass):
     registry_.save()
 
     count = 0
+    previously_seen_instrument = []
     # traverse directory of files
     for dataset in datasets:
```

Comment:
Likewise, rename `dataset` to `dataset_path` and the rest of the for loop block.